docs/api/v1/fluxinstance.md

```diff
@@ -23,6 +23,7 @@ metadata:
   annotations:
     fluxcd.controlplane.io/reconcile: "enabled"
     fluxcd.controlplane.io/reconcileEvery: "1h"
+    fluxcd.controlplane.io/reconcileArtifactEvery: "10m"
```

Comment:
We don't set the artifact in this example, so need to remove the annotation here too.

Reply:
fixed